app/config/parameters.yml.dist

```diff
@@ -14,7 +16,5 @@ parameters:
     mailer_user:       ~
     mailer_password:   ~
 
-    locale:            en
-
     # A secret key that's used to generate certain security-related tokens
     secret:            ThisTokenIsNotSoSecretChangeIt
```

Comment:
As we discussed in https://github.com/symfony/symfony/issues/14026 the secret MUST NOT be different for each machine. So this stays in contrast to the added comment at the top of the file.